src/transport/tests/TestCryptoContext.cpp

```diff
@@ -72,4 +72,11 @@ TEST_F(TestGroupCryptoContext, TestBuildPrivacyNonce)
     }
 }
 
+TEST_F(TestGroupCryptoContext, TestEncryptionOverhead)
+{
+    // Matter has a single set of crypto primitives and thus a fixed amount of overhead.
```

Comment:
```suggestion
    // This may seem trivial, but adding new crypto suite should lead to
    // an update of this test to properly vet the overhead for all suites.
```